test/sdf/pose_1_9.sdf

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" ?>
+<sdf version="1.9">
+  <world name="default">
+
+    <model name="model_with_empty_pose">
+      <pose></pose>
+      <link name="link"/>
+    </model>
+
+    <model name="model_with_empty_pose_with_degrees">
+      <pose rotation_type="rpy_degrees"></pose> 
+      <link name="link"/>
+    </model>
+
+    <model name="model_with_empty_pose_with_quaternion">
+      <pose rotation_type="q_wxyz"></pose> 
+      <link name="link"/>
+    </model>
+
+    <model name="model_with_rpy_pose_no_attribute">
+      <pose>1 2 3   0.4 0.5 0.6</pose>
+      <link name="link"/>
+    </model>
+
+    <model name="model_with_rpy_pose_with_degrees">
```

Comment:
nit The relation among these different pose representations is unclear.

Consider:
- Adding a comment saying all poses are unrelated, or
- Change all non-identity poses to represent the same orientation, e.g. 90 degrees about x-axis or what not.

Reply:
Added a comment here, https://github.com/ignitionrobotics/sdformat/pull/589/commits/002cc42c41d8b36115b2ed8752b73515e5dc4113, let me know if this is what you meant, thanks!

Reply:
Yup, perfect!